crates/holochain_zome_types/src/query.rs

```diff
@@ -78,7 +80,7 @@ pub struct ChainQueryFilter {
     // NB: if this filter is set, you can't verify the results, so don't
     //     use this in validation
     pub action_type: Option<Vec<ActionType>>,
-    /// Include the entries in the records
+    /// Include the entries in the records. This value is ignored for [ `get_agent_activity` ].
     pub include_entries: bool,
```

Comment:
Huh, was this intentional design? Should it not be ignored?

Reply:
gooooooooooood question. It's unsupported all the way to the DHT op itself; check out [`ChainOp::RegisterAgentActivity`](https://docs.rs/holochain_types/latest/holochain_types/dht_op/enum.ChainOp.html#variant.RegisterAgentActivity). So it's not so much that `get_agent_activity` ignores it; it's that it can't do anything with it.